portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,31 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Lorna Armstrong's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>Welcome to the porfolio of Lorna Armstrong.</p>
+
+      <h3>About Me</h3>
+      <p>My name is Lorna Armstrong and I am from Middlesbrough, England. 
+          I am a Computer Science student at the University of Edinburgh, 
+          heading into my third year in September 2020. Currently, I am 
+          working as a STEP Intern for Google, working remotely from Edinburgh.
+          This portfolio will show you a little bit about me.</p>
+
+      <img src= "/images/profile.jpg" alt="Profile Picture" style="width:200px;height:200px;">    
+      <img src= "/images/edinburgh.jpg" alt="Skyline of Edinburgh" style="width:400px;height:250px;">
+      <img src= "/images/skye.jpg" alt="My Border Collie, Skye" style="width:300px;height=300px;">
```

Comment:
Styles usually go to CSS files and that will help you to keep your HTML clean (using CSS is [optional](https://github.com/googleinterns/step/blob/master/walkthroughs/week-2-web-development/portfolio-walkthrough.md#css) though). However, you don't even need `style` here, `img` tag already has got `height` and `width` attributes ([ref](https://www.w3schools.com/tags/att_img_width.asp)) which would be more appropriate here I think.

Reply:
Thanks for this feedback, Kazi. I didn't realise that img already has the attributes I needed, it's really useful to know!